### ISSUE TITLE
Discussion about the Dashboard link on admin page titles

### ISSUE THREAD

**Issue by GrCOTE7** [bestmomo/sillo#63]
Hi!
I noticed that in the commit of 07/18/24 - 00:47, code setting, the bulk of the version was to remove the link for the dashboard placed on the page titles...

Well yes, when we're on a large screen, it's also right there in the admin menu bar, visible...

But on small screens? And since we're often needing to return to the dashboard, it avoids having to open the side menu, instead having a link right there, direct :-)
(The User admin page still have it...)

In short, this link doesn't cost much, and it helps improve the UX ;-)

No ;-) ?

**Comment by bestmomo** [comment]
A link to the Dashbord may indeed be relevant on a small screen, although using the administration on a phone doesn't seem very sensible to me. But the link in the page title wasn't really intuitive, so we might as well add a button.

**Comment by GrCOTE7** [comment]
Yes, that can be a very good compromise :-) !

**Comment by GrCOTE7** [comment]
And you done it very well ! Thanks !

(Of course, to apply at all admin pages... ;-)...)
